components/common/WalletModal.vue

```diff
@@ -61,10 +61,19 @@
 
         <div v-if="hasSelectedWalletProvider && hasWalletProviderExtension">
           <div class="subtitle has-text-centered">Choose your account</div>
-          <AccountSelect
-            v-model="account"
-            :label="$i18n.t('Account')"
-            :tooltip-visible="false" />
+
+          <b-field :label="$i18n.t('Account')">
```

Comment:
Maybe move it to a separated component?

Reply:
actually that AccountSelect component, I've tried to edit it so I can pass accounts list as props but something wasn't working